imap_processing/tests/codice/test_codice_hi_l2.py

```diff
@@ -45,23 +43,23 @@ def test_l2_hi_omni(mock_get_file_paths, codice_lut_path):
             err_msg=f"Mismatch in variable '{variable}'",
         )
 
+    processed_l2.attrs["Data_version"] = "001"
+    omni_cdf_file = write_cdf(processed_l2)
+    assert omni_cdf_file.name == "imap_codice_l2_hi-omni_20250814_v001.cdf"
```

Comment:
Won't this get automatically filled in for you so you don't have to modify any attributes? Similar below.
```suggestion
    omni_cdf_file = write_cdf(processed_l2)
    assert omni_cdf_file.name == "imap_codice_l2_hi-omni_20250814_v999.cdf"
```

Reply:
I am writing certain version in case Joey found something I need to update and I can update to newer version instead of always producing v999.